dev/docker-compose.yaml

```
@@ -69,6 +69,20 @@ services:
       - wp_deploy
     ports:
       - 3306:3306
+  phpmyadmin:
+    image: phpmyadmin/phpmyadmin
+    links:
+      - db
+    environment:
+      PMA_HOST: db
+      PMA_PORT: 3306
+      PMA_ARBITRARY: 1
+      PMA_USER: root
+      PMA_PASSWORD: ${MYSQL_ROOT_PASSWORD}
+    networks:
+      - wp_deploy
+    ports:
+      - 8081:80
```

Sometimes this would be a good thing to add as an env variable.  And the only reason i would do this, is you don't know what ports developers uses...
```suggestion
      - ${MYADMIN_PORT:-8081}:80
```

---

Good point. Added your suggestion.